sql/core/src/main/scala/org/apache/spark/sql/execution/joins/HashedRelation.scala

```diff
@@ -923,9 +1006,9 @@ case class HashedRelationBroadcastMode(key: Seq[Expression])
       sizeHint: Option[Long]): HashedRelation = {
     sizeHint match {
       case Some(numRows) =>
-        HashedRelation(rows, canonicalized.key, numRows.toInt)
+        HashedRelation(rows, canonicalized.key, numRows.toInt, isNullAware = isNullAware)
```

Comment:
nit: Should this simply be `isNullAware` (ie passed in without a named argument)

Reply:
there is still a taskMemoryManager: TaskMemoryManager = null (with default value) just before isNullAware, I've had to using the named argument. ^_^